javascript.js

```diff
@@ -1,3 +1,46 @@
-//let text = 'Hello World!'
-//alert(text)
-//console.log(text)
+/* Befehl für den Zugriff auf die jeweiligen id-Elemente + 
+Zuweisung auf eine Variable. Hier: main-Elemente */
+const questionContent = document.querySelector('#questions')
+const bookmarksContent = document.querySelector('#bookmarks')
+const createContent = document.querySelector('#create')
+const profileContent = document.querySelector('#profile')
+
+/*Befehl für den Zugriff auf die jeweiligen id-Elemente + 
+Zuweisung auf eine Variable. Hier: button-Elemente im footer*/
+const questionButton = document.querySelector('#question-button')
+const bookmarkButton = document.querySelector('#bookmark-button')
+const createButton = document.querySelector('#create-button')
+const profileButton = document.querySelector('#profile-button')
+
+questionButton.addEventListener('click', () => {
+  questionContent.classList.remove('main--hidden')
+  bookmarksContent.classList.add('main--hidden')
+  createContent.classList.add('main--hidden')
+  profileContent.classList.add('main--hidden')
+})
+
+/*EventListener für Click Aktion +
+Hidden  question-main
+Sichtbarkeit bookmark-main
+Hidden create-main
+*/
+bookmarkButton.addEventListener('click', () => {
+  questionContent.classList.add('main--hidden')
+  bookmarksContent.classList.remove('main--hidden')
+  createContent.classList.add('main--hidden')
+  profileContent.classList.add('main--hidden')
+})
+
+createButton.addEventListener('click', () => {
+  questionContent.classList.add('main--hidden')
+  bookmarksContent.classList.add('main--hidden')
+  createContent.classList.remove('main--hidden')
+  profileContent.classList.add('main--hidden')
+})
+
+profileButton.addEventListener('click', () => {
+  questionContent.classList.add('main--hidden')
+  bookmarksContent.classList.add('main--hidden')
+  createContent.classList.add('main--hidden')
+  profileContent.classList.remove('main--hidden')
+})
```

Comment:
Small hint: a better naming convention for this file would be `main.js` or `script.js` instead of `javascript.js`.

Reply:
Thanks! :slightly_smiling_face: 
And yes, thanks for the hint. I took notes when Lukas mentioned it, but haven't changed it for the quiz app :joy: